crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/controller/CrownCourtContributionController.java

```diff
@@ -33,7 +33,7 @@ public class CrownCourtContributionController {
     @Operation(description = "Calculate Contribution")
     @ApiResponse(responseCode = "200",
             content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
```

Comment:
Could you please change the rest uri naming convention e.g: /api/internal/v1/contribution/get-contribution-summaries to /api/internal/v1/contribution/summaries